lib/pf/config/cached.pm

```diff
@@ -835,6 +916,23 @@ sub cacheForData {
     return pf::CHI->new(namespace => 'configfilesdata' );
 }
 
+=head2 RefreshConfigs
+
+refresh configs and call any register callbacks 
+
+=cut
+
+sub RefreshConfigs {
+    my $logger = get_logger();
+    $logger->trace("Started Reloading all configs");
+    foreach my $config (@LOADED_CONFIGS{@LOADED_CONFIGS_FILE}) {
+        next unless $config;
+        next unless $config->LockFileHasChanged;
+        $config->ReloadConfig();
+    }
+    $logger->trace("Finished Reloading all configs");
+}
```

Comment:
Your log entries for this sub are the same as for `ReloadConfigs`. You should reword them.